vignettes/web_only/repFilter_v3.Rmd

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Data Filtering'
+author: '<a href="https://immunomind.io">ImmunoMind</a>'
+date: "support@immunomind.io"
+output:
+  html_document:
+    fig_height: 8
+    fig_width: 10
+    theme: spacelab
+    toc: yes
+  pdf_document:
+    toc: yes
+  word_document:
+    toc: yes
+---
+
+
+<!--
+%\VignetteEngine{knitr::rmarkdown}
+%\VignetteIndexEntry{Gene usage analysis}
+%\VignettePackage{immunarch}
+-->
+
+
+
+```{r setup, include=FALSE, echo=FALSE}
+# knitr::knit_hooks$set(optipng = knitr::hook_optipng)
+# knitr::opts_chunk$set(optipng = '-o7')
+
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(fig.align = "center")
+knitr::opts_chunk$set(fig.width = 12)
+knitr::opts_chunk$set(fig.height = 6)
+
+library(immunarch)
+```
+
+# Data filtering
+For filtering your data you can use universal function `repFilter`. 
+
+## Methods for filtering data
+`repFilter` has 2 parameters: `method ` and `query`.
```

Comment:
In my mind, there should be .method and .query parameters to save naming conventions of Immunarch

Reply:
Also, there should be .match argument